FixedFormPackager/Program.cs

```diff
@@ -65,6 +66,24 @@ private static void Main(string[] args)
                             Directory.CreateDirectory(args[i]);
                         }
                             break;
+                        case "-id":
```

Comment:
Let's switch to use https://github.com/gsscoder/commandline for the command line parsing.  i've used it before and much easier to understand and code